### ISSUE TITLE
New prediction on my files

### ISSUE THREAD

**Issue by babak-abad** [experiencor/keras-yolo3#232]
I train the model on 300x60 images for character localization but I stop train manually. The loss of xy, wh were less than 2. Now I test the model using
`!python predict.py -c config.json -i '/content/gdrive/My Drive/Colab Notebooks/keras-yolo3-master/keras-yolo3-master/test'`

The test folder contains more than 100 images sized to 300x60.
no bounding box for my images were found. What is the problem? Any help will be appreciated...

**Comment by babak-abad** [comment]
Also I test:
!python predict.py -c config.json -i '/content/gdrive/My Drive/Colab Notebooks/keras-yolo3-master/keras-yolo3-master/test/**myFileName.bmp**'

**Comment by SteveIb** [comment]
I think there is something wrong with bmp files, it seems it needs rgb color not grayscale. I'm facing the same issue!
    def _aug_image(self, instance, net_h, net_w):
        print(instance['filename'])
        image_name = instance['filename']
        print('image name is ',image_name)
        if os.path.exists(image_name):
            print('path exist')
        image = cv2.imread(image_name,cv2.IMREAD_GRAYSCALE) # RGB image

it is just a hint.. i edited the code.

**Comment by SteveIb** [comment]
some opencv versions have issues reading bmp files.

https://stackoverflow.com/questions/52783150/imread-still-returns-empty-mat-despite-correct-addressing-in-opencv-4-0-0
"Thank you for your answering. I just uninstalled openCV 4.0.0 and installed again in version 3.4.1, then it worked well. Maybe there is something wrong with version 4.0.0 because it is alpha version yet. But I don't know exactly the reason why it didn't work. Anyway, I appreciate to your kindness :) "

So My solutions was to convert all bmp files to jpg files then do training!
